README.md

```diff
@@ -1,28 +1,24 @@
-# Thermal Bridging & Derating (TBD)
```

Comment:
@macumber: With the online [Guide to TBD](https://rd2.github.io/tbd), no point keeping as much info in the README.